src/general/OptionsMenu.tscn

```diff
@@ -290,6 +290,8 @@ size_flags_horizontal = 3
 custom_minimum_size = Vector2(200, 25)
 layout_mode = 2
 size_flags_vertical = 0
+focus_neighbor_top = NodePath("../../Fullscreen")
```

Comment:
As far as I can tell focus moves correctly even without these. To avoid massive amounts of properties that need maintaining when the GUI slightly changes, I do not add explicit focus neighbours when the automatic detection works correctly.